packages/beacon-node/src/chain/bls/multithread/index.ts

```diff
@@ -357,25 +411,42 @@ export class BlsMultiThreadWorkerPool implements IBlsVerifier {
   }
 
   /**
-   * Potentially submit jobs to an idle worker, only if there's a worker and jobs
+   * Prepare work and send to either workerpool or to libuv threadpool
    */
   private runJob = async (): Promise<void> => {
     if (this.closed) {
       return;
     }
+    // Prepare work package
+    const prepared = this.prepareWork();
+    if (prepared.length === 0) {
+      return;
+    }
+
+    if (this.blsPoolType === BlsPoolType.workers) {
+      await this.runJobWorkerPool(prepared);
+    } else {
+      await this.runJobLibuv(prepared);
+    }
 
+    // Potentially run a new job
+    setTimeout(this.runJob, 0);
+  };
+
+  private runJobLibuv = async (jobs: JobQueueItem[]): Promise<void> => {
+    await this._runJob(jobs, asyncVerifyManySignatureSets);
+  };
+
+  /**
+   * Potentially submit jobs to an idle worker, only if there's a worker and jobs
```

Comment:
add comment noting this is deprecated from v1.18, will be cleaned up later

Reply:
Updated.